Add note on upgrade paths for NOM (#82)
neo4j/docs-ops-manager: modules/ROOT/pages/update/index.adoc

= Updating to a newer version
:description: This section describes the upgrade process from an earlier version of NOM.

> It is recommended to always keep NOM up-to-date with the latest available version. You can update from any older NOM version to any newer version. A version downgrade is not possible. The natural order of NOM versions follows our

Reviewer: ```suggestion
It is recommended to always keep NOM up-to-date with the latest available version. 
You can update from any older NOM version to any newer version. 
A version downgrade is not possible. 
The natural order of NOM versions follows the *xref:../appendix/version-compatibility.adoc#versioning_scheme[semantic versioning scheme]*.
```

> It is recommended to always keep NOM up-to-date with the latest available version. You should be able to update from any older NOM version to any newer version. A version downgrade is not possible. The natural order of NOM versions follows our

Reviewer 2: ```suggestion
It is recommended to always keep NOM up-to-date with the latest available version. You can update from any older NOM version to any newer version. A version downgrade is not possible. The natural order of NOM versions follows our
```

Reviewer 2: Let's project some more confidence here! 😄

> *xref:../appendix/version-compatibility.adoc#versioning_scheme[semantic versioning scheme]*.

Reviewer: ```suggestion

```
